tests/test_saucedemo_e2e_tests.py

```diff
@@ -32,10 +35,57 @@ def test_success_login(self) -> None:
         password_input.click()
         password_input.send_keys(CORRECT_PASSWORD)
 
-        login_button = self.driver.find_element(By.ID, LOGIN_BUTTON_LOCATOR_ID)
-        login_button.click()
+        login_button = self.driver.find_element(By.ID, LOGIN_BUTTON_LOCATOR_ID).click()

```

Comment:
in new line of code you saving into variable login_button None, which is returned by method click(), and login_button is not used later on
please remove variable or revert to previous code

Reply:
sure thing, thx